### PR TITLE
feat: BASH_SHFMT linter + linter fixes

### PR DESCRIPTION
**Pull request by javydekoning**
# Why?

For consistent formatting of bash scripts. 

*Issue #, if available:*

## What?

Added BASH_SHFMT

---

By submitting this pull request, I confirm that you can use, modify, copy, and
redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: samples/sample-ec2-java-app-codedeploy/scripts/start.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 cd /home/ec2-user/server
 sudo /usr/bin/java -jar -Dserver.port=80 \
-  *.jar > /dev/null 2> /dev/null < /dev/null &
+  *.jar >/dev/null 2>/dev/null </dev/null &
```
Comment (sbkok) [inline samples/sample-ec2-java-app-codedeploy/scripts/start.sh:5]: Might be me, but personally I like the space after the `>` sign.
It makes things more readable I think. What are your thoughts?
And can we change this setting?

A bit surprising that it does enforce spaces around `|`.

File: samples/sample-ec2-with-codedeploy/scripts/install-deps.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ echo "<VirtualHost *:80>
   ProxyRequests Off
   ProxyPass / http://localhost:8080/
   ProxyPassReverse / http://localhost:8080/
-</VirtualHost>" >> sudo /etc/httpd/conf/httpd.conf
+</VirtualHost>" /etc/httpd/conf/httpd.conf >>sudo
```
Comment (sbkok) [inline samples/sample-ec2-with-codedeploy/scripts/install-deps.sh:37]: This would not work I think, would it?

**Review by sbkok — DISMISSED** [review]

File: samples/sample-ec2-with-codedeploy/scripts/install-deps.sh (hunk 2 above)
Comment (javydekoning) [reply samples/sample-ec2-with-codedeploy/scripts/install-deps.sh:37]: That does look odd... Let me move this to draft and investigate

Comment (javydekoning) [reply samples/sample-ec2-with-codedeploy/scripts/install-deps.sh]: I modified this as follows:

```sh
# Append VirtualHost configuration
echo "
<VirtualHost *:80>
  ProxyRequests Off
  ProxyPass / http://localhost:8080/
  ProxyPassReverse / http://localhost:8080/
</VirtualHost>
" | sudo tee -a /etc/httpd/conf/httpd.conf >/dev/null
```

File: samples/sample-ec2-java-app-codedeploy/scripts/start.sh (hunk 1 above)
Comment (javydekoning) [reply samples/sample-ec2-java-app-codedeploy/scripts/start.sh:5]: I agree with you. Adding `space_redirects = true` to editorconf to fix this.

**Review by sbkok — APPROVED** [review]
LGTM, thank you!

**Review by StewartW — APPROVED** [review]

**Pull request merged**